### ISSUE TITLE
text fields validation limit to 255 chars is not longer required

### ISSUE THREAD

**Issue by weiyiwang-dotcms** [dotCMS/core#26774]
### Problem Statement

When adding text to a text field in a contentlet, dotCMS enforces a 255 character limit for any content type. This affects any users who would like to add longer text information to content types, particularly for vanity URLs, where the URI and destination fields are already defined as text fields.

### Steps to Reproduce

Add a new vanity URL
Try adding a long URI or long destination that is greater than 255 characters in length.
E.g
```
this-text-is-waaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaay-longer-than-two-hundred-fifty-five-characters-so-should-be-hitting-the-limit-for-the-fields-260-charsq
```
You will receive a popup modal that states you cannot add more than 255 characters

### Acceptance Criteria

Remove the character limit validation for fields

### dotCMS Version

Versions post implementation of contentlet_as_json

### Proposed Objective

Technical User Experience

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

https://dotcms.zendesk.com/agent/tickets/114171

https://dotcms.slack.com/archives/G01N0AU1NUC/p1700502221199139

### Assumptions & Initiation Needs

The 255 character limit should no longer apply, since contentlet_as_json can store much more than 255 characters.

**Comment by binarybasher** [comment]
Would this be solved by editing the "Vanity URL" Content Type and replacing the Text fields (URI, forwardTo) with TextArea fields?

**Comment by binarybasher** [comment]
We hit this issue with importing Vanity URLs, however no error was thrown.

**Comment by binarybasher** [comment]
Thanks for confirming @erickgonzalez 
It was worth a try, we have many Vanity URL's unable to work due to this and the Cyrillic issue.

**Comment by binarybasher** [comment]
It's not clear to me if this was resolved in 23.01.9 or if it's still open?

**Comment by erickgonzalez** [comment]
Hi @binarybasher,  this is resolved in 23.01.9 but the ticket is still open since we also fixed it in the master branch, and needs to be QA over there.

**Comment by josemejias11** [comment]
**Approved**: Tested on **master_15e8ed4**, Docker, macOS 13.0, FF v113.0
